daemon/tests/harness/mod.rs

```diff
@@ -310,15 +325,23 @@ macro_rules! deliver_event {
     ($maker:expr, $taker:expr, $event:expr) => {{
         tracing::debug!("Delivering event: {:?}", $event);
 
-        $taker.system.cfd_actor_addr.send($event).await.unwrap();
-        $maker.system.cfd_actor_addr.send($event).await.unwrap();
+        $taker.system.cfd_actor.send($event).await.unwrap();
+        $maker.system.cfd_actor.send($event).await.unwrap();
     }};
 }
 
 pub fn dummy_price() -> Price {
     Price::new(dec!(50_000)).expect("to not fail")
 }
 
+pub fn dummy_quote() -> Quote {
+    Quote {
+        timestamp: Timestamp::now(),
+        bid: Price::new(dec!(50_000)).expect("to not fail"),
+        ask: Price::new(dec!(50_000)).expect("to not fail"),
```

Comment:
nit: would probably use the `dummy_price()` inside dummy quote, no need to have more magic numbers than necessary